examples/hello-world/format-example.cpp

```diff
@@ -0,0 +1,130 @@
+
+#include "internalGroup1.h"
+#include "slang.h"
+#include <algorithm>
```

Comment:
I'd advocate for `SortIncludes: true`, since we mostly use that style today. I think keeping `IncludeBlocks: Preserve` gives us the flexibility to override the ordering in cases where we care.

As long as we are at it, I'd also advocate for `SortUsingDeclarations: true`.